winui/MaskedTextBox/Overview.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: About WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI MaskedTextBox(SfMaskedTextBox) Control with essential features and more here.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# WinUI MaskedTextBox Overview
+
+The Masked text box is an advanced version of the Entry control that restricts your input to certain types of characters, text, and numbers using a mask pattern. This control is used to create a template for providing information such as telephone numbers, IP addresses, product IDs, and so on.
+
+![MaskedTextBox control overview in WinUI](MaskedTextBox_Images/winui_maskedtextbox_overview.png)
```

Comment:
@Vignesh-SF3580 Use the below content.

The .NET WinUI SfMaskedTextBox is an advanced version of the input control that restricts input of certain characters, text, and numbers by using a mask pattern. This control is used to create a template for providing information such as telephone numbers, email IDs, IP addresses, product keys, and so on. 

### Key features

* **MaskTypes**: The input can be masked with a fixed or variable length by setting the mask type to simple or regex, respectively.  
* **PromptChar**: Customize the prompt characters used to set the default characters as input in the absence of input. 
* **Value**: Enter values and clipboard operations in the control based on the ValueMaskFormat property.
* **ValueMaskFormat**: Setting formatted values in the control using mask format values such as prompt, literals, and both.
* **Customization**: Customize the UI of the masked text box.

Reply:
Content changed like above.